### DISCUSSION TITLE
How to float stats card to some text content right side?

### DISCUSSION THREAD

**Discussion by recherst** [anuraghazra/github-readme-stats#2202] [Q&A] [answered]
Somebody asked how to align image to right side, and relational answer are here: [#256](https://github.com/anuraghazra/github-readme-stats/issues/256).
But, if I use `align=right` , the image is wrapped in new line, I want it float to left texts right side.
How? I've been tried many times, but it not worked. 
Help me, please!

**Comment by rickstaa** [comment, marked as answer]
@recherst Thanks for your question. The syntax explained in #256 should work:

```html
<a href="https://github.com/anuraghazra/github-readme-stats">
  <img align="left" src="https://github-readme-stats.vercel.app/api/pin/?username=anuraghazra&repo=github-readme-stats" />
</a>
<a href="https://github.com/anuraghazra/convoychat">
  <img align="left" src="https://github-readme-stats.vercel.app/api/pin/?username=anuraghazra&repo=convoychat" />
</a>
```

You can checkout [my profile](https://github.com/rickstaa/rickstaa) for an example.
